torch/csrc/jit/tensorexpr/kernel.h

```diff
@@ -88,7 +88,8 @@ std::vector<T> convertVecArgValue(const std::vector<ArgValue>& v) {
 class TORCH_API TensorExprKernel {
   struct ConstantDescr {
     BufPtr buf;
-    void* ptr;
+    void* ptr = nullptr;
+    Node* node = nullptr;
```

Comment:
Could you please add a comment about these fields? I.e. that only one of them is expected to be non-null and what each of these situations means.